src/Surfnet/Stepup/Identity/Identity.php

```diff
@@ -979,6 +1002,10 @@ protected function applyIdentityForgottenEvent(IdentityForgottenEvent $event)
         $this->forgotten = true;
     }
 
+    protected function applyVettedSecondFactorsAllRevokedEvent(VettedSecondFactorsAllRevokedEvent $event)
+    {
+        $this->vettedSecondFactors->clear();
```

Comment:
Is this handler required in broadway? i don't think the body is required right?

Reply:
That shouldn't be necessary  indeed, so removed it.